apps/proposer/src/proposer.mjs

```diff
@@ -14,13 +11,15 @@ export default async function startProposer(nf3, proposerBaseUrl) {
   // Mnemonic are only required for services connecting to a client that
   // can generate a compressed PKD.
 
-  await nf3.init(undefined, 'optimist');
+  await nf3.init(
```

Comment:
propose doesnt need a mnemonic for now. And if it did, it may be better with a variable